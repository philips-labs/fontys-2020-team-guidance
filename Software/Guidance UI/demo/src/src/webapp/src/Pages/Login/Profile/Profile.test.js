import React from 'react';
import { cleanup } from '@testing-library/react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Profile from "./Profile";


afterEach(cleanup);

it('renders correctly react-test-renderer', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Profile />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
});



