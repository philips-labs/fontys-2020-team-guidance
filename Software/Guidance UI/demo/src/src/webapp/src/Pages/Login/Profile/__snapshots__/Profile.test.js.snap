// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly react-test-renderer 1`] = `
<div
  className="container"
>
  <header
    className="jumbotron"
  >
    <h3
      style={
        Object {
          "color": "#FFFFFF",
        }
      }
    >
      
       Profile
    </h3>
  </header>
  <p
    style={
      Object {
        "color": "#FFFFFF",
      }
    }
  >
    Email: 
    
  </p>
</div>
`;
